### PR TITLE
Added JEST/ENZYME tests for components

### PR DESCRIPTION
**Pull request by dcheng1290**
Please review my sample testing files for my components and I used Jest/Enzyme for them. All tests have passed via terminal when i run npm run test. You can ignore all other files other than the test files.

The About.test.js is testing <App/> component as I did not merge my About.jsx yet. It will be updated to About.jsx once I merge my other pull request.

### CODE REVIEW

**Comment by dcheng1290** [comment]
[PR David's basic jest/enzyme test files for react components](https://trello.com/c/IE9typHV/44-pr-davids-basic-jest-enzyme-test-files-for-react-components)

File: tests/about.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import renderer from 'react-test-renderer';
+import App from '../client/src/App.jsx';
+import { shallow, mount, render } from 'enzyme';
+
+describe('App component renders correctly', () => {
+  it('renders correctly', () => {
```
Comment (careyl96) [inline tests/about.test.js]: change to it('should render correctly')

File: tests/app.test.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+import renderer from 'react-test-renderer';
+import App from '../client/src/App.jsx';
```
Comment (careyl96) [inline tests/app.test.js:4]: import components last

File: tests/comment.test.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import renderer from 'react-test-renderer';
+import Comment from '../client/src/Comment.jsx';
```
Comment (careyl96) [inline tests/comment.test.js]: import components last

File: tests/comment.test.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  fullName: 'David Cheng',
+  userImage: 'https://s3.amazonaws.com/uifaces/faces/twitter/tristanlegros/128.jp',
+  commentsText: 'Labore sed dicta amet aut recusandae. Voluptates dolorem non. Tempora vel possimus voluptas. Qui sunt et sint dolor impedit. Ratione quos quia iste eum temporibus sit. Illum et minima quibusdam sint laudantium cum. Voluptatem quod quam et odio quaerat et minima dignissimos magnam. Optio et impedit et eum sit. Enim praesentium et quia. Nihil neque quos in et.',
+};
+
+describe('Comment Component', () => {
+  it('should render without throwing an error', () => {
+    expect(shallow(<Comment comment={fakeComment} />));
+  });
+  it('correctly renders info section', () => {
```
Comment (careyl96) [inline tests/comment.test.js]: include spaces after each block

**Review by careyl96 — CHANGES_REQUESTED** [review]
Looks good other than a few things here and there. Fix those up and we'll be good to go

**Comment by dcheng1290** [comment]
Updated import orders and spacing after blocks. Thanks.

**Review by careyl96 — DISMISSED** [review]
looks good!

**Review by careyl96 — APPROVED** [review]

**Pull request merged**